consensus/processor.go

```diff
@@ -280,6 +282,13 @@ func (p *DefaultProcessor) ApplyMessage(ctx context.Context, st state.Tree, vms
 		return nil, errors.FaultErrorWrap(err, "could not get message cid")
 	}
 
+	ctx, err = tag.New(ctx,
+		tag.Insert(msgMethodKey, msg.Method),
```

Comment:
Messages without a method name represent sending filecoin from one actor to another, should we adjust the tag to reflect this, e.g.:
```go
if msg.Method == "" {
    tagString = "send"
}
```

Reply:
SGTM.  Maybe call it "sendFIL" to differentiate between general vm message sending.